src/couch_replicator/src/couch_replicator_auth_session.erl

```diff
@@ -170,11 +176,13 @@ handle_call(stop, _From, State) ->
 
 
 handle_cast(Msg, State) ->
+    ?LOG_ERROR(#{what => unexpected_cast, in => replicator, msg => Msg}),
     couch_log:error("~p: Received un-expected cast ~p", [?MODULE, Msg]),
     {noreply, State}.
 
 
 handle_info(Msg, State) ->
+    ?LOG_ERROR(#{what => unexpected_msg, in => replicator, msg => Msg}),
```

Comment:
Nitpick.

It is better to be consistent the `unexpected_message` is used in `couch_js_os_process`.